packages/fx-core/templates/plugins/resource/aad/bicep/variables.template.bicep

```diff
@@ -0,0 +1,14 @@
+
+{{#contains 'fx-resource-frontend-hosting' plugins}}
+{{#notContains 'fx-resource-bot' ../plugins}}
+var applicationIdUri = `api://{{../fx-resource-frontend-hosting.outputs.domain}}/${AADClientId}`
+{{/notContains}}
+{{#contains 'fx-resource-bot' ../plugins}}
+var applicationIdUri = `api://{{../fx-resource-frontend-hosting.outputs.domain}}/botid-${BotClientId}`
+{{/contains}}
+{{/contains}}
+{{#notContains 'fx-resource-frontend-hosting' plugins}}
+{{#contains 'fx-resource-bot' ../plugins}}
+var applicationIdUri = `api://{{../fx-resource-bot.outputs.domain}}/botid-${BotClientId}`
```

Comment:
only need to include the `botid-${BotClientId}` part

Reply:
Do you mean 
```
var applicationIdUri = `api:///botid-${BotClientId}`
```

Reply:
yes

Reply:
Fixed.